### PR TITLE
added env file to the intstall dir, changed order of execution of env…

### PR DESCRIPTION
**Pull request by jawhnycooke**
… file, added -y to apt commands

### CODE REVIEW

File: installation/env [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Go related variables, shouldn't need to be changed
+export GOPATH=$HOME/asic_q2
```
Comment (sfloresk) [inline installation/env]: asic_q2 is not representative to the project. Can you change it to ztp_go or something similar please?. Thank you

File: installation/setup.sh [hunk 2]
```diff
@@ -44,8 +44,9 @@ source /etc/profile
 
 #setup go files
 mkdir $HOME/asic_q2
+cp env $HOME/asic_q2/.env
```
Comment (sfloresk) [inline installation/setup.sh]: asic_q2 is not representative to what the solution does. See comment above

**Review by sfloresk — CHANGES_REQUESTED** [review]
Please see inline comments related to asic_q2 directory name. Thanks!

File: installation/env (hunk 1 above)
Comment (jawhnycooke) [reply installation/env]: will do

File: installation/setup.sh (hunk 2 above)
Comment (jawhnycooke) [reply installation/setup.sh]: will do

**Comment by jawhnycooke** [comment]
made edits based on convo with devs

**Pull request merged**